kubernetes/cmsweb/services/reqmgr2ms-output.yaml

```diff
@@ -1,25 +1,25 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: reqmgr2ms-o
+  name: ms-output
   namespace: dmwm
 spec:
   selector:
-    app: reqmgr2ms-o
+    app: ms-output
   ports:
-    - port: 8248
-      targetPort: 8248
-      name: reqmgr2ms-o
+    - port: 8245
+      targetPort: 8245
+      name: ms-output
     - port: 18248
```

Comment:
don't you want to update the port used by prometheus to collect service metrics as well?